### DISCUSSION TITLE
TypeError: VegaChartsMixin.line_chart() got an unexpected keyword argument 'caption'

### DISCUSSION THREAD

**Discussion by emersonberghe** [community/community#127125] [Programming Help]
### Select Topic Area

Bug

### Body

O Código a seguir retorna o erro:
import streamlit as st

# Sample data (replace with your actual data acquisition)
data = [10, 20, 30, 40]

st.line_chart(data, caption="Sample Line Chart")

if __name__ == "__teste__":
    # Run the Streamlit app
    st.sidebar.title("Line Chart Example")
    st.balloons()  # Optional: Add a balloon to attract attention

Erro retornado:
C:\Users\emers\OneDrive\Documentos\cursos\Python\pythonProject\CotacaoAcoes2\.venv\Scripts\python.exe C:\Users\emers\OneDrive\Documentos\cursos\Python\pythonProject\CotacaoAcoes2\teste.py 
Traceback (most recent call last):
  File "C:\Users\emers\OneDrive\Documentos\cursos\Python\pythonProject\CotacaoAcoes2\teste.py", line 6, in <module>
    st.line_chart(data, caption="Sample Line Chart")
  File "C:\Users\emers\OneDrive\Documentos\cursos\Python\pythonProject\CotacaoAcoes2\.venv\lib\site-packages\streamlit\runtime\metrics_util.py", line 397, in wrapped_func
    result = non_optional_func(*args, **kwargs)
TypeError: VegaChartsMixin.line_chart() got an unexpected keyword argument 'caption'

Process finished with exit code 1

**Comment by 0xpuck** [comment]
To fix the issue, you should remove the caption argument from the st.line_chart function. The reason for the error is the line_chart method in Streamlit not accepting a caption argument.

This is the updated code:
```
import streamlit as st

data = [10, 20, 30, 40]

st.line_chart(data)

if __name__ == "__main__":
    # Run the Streamlit app
    st.sidebar.title("Line Chart Example")
    st.balloons()
```
I hope this is you to help
